rustls/src/client/tls13.rs

```diff
@@ -642,9 +654,9 @@ impl hs::State for ExpectCertificateVerify {
             .split_first()
             .ok_or(Error::NoCertificatesPresented)?;
         let now = std::time::SystemTime::now();
-        let cert_verified = cx
+        let cert_verified = self
             .config
-            .get_verifier()
+            .verifier
```

Comment:
Not sure why we're removing `get_verifier()` here. Seems like an orthogonal change.

Reply:
Added it back.